lib/xml/parser.js

```diff
@@ -99,6 +99,8 @@ function parseEvents(xml) {
         });
 
         return formatted;
+    }).catch(function (err) {
```

Comment:
I think it would be better to pass this error up to the code which uses this library except if we can handle it somehow within scrapgoat.